### PR TITLE
Spring hw

### PR DESCRIPTION
**Pull request by margarita-ignatieva**

### CODE REVIEW

File: src/main/java/dao/impl/UserDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        }
+    }
+
+    @Override
+    public List<User> listUsers() {
+        log.info("Trying to find all users");
+        try (Session session = sessionFactory.openSession()) {
+            Query<User> query = session.createQuery("from User", User.class);
+            List<User> users = query.getResultList();
+            return users;
```
Comment (KostiantynVorobiov) [inline src/main/java/dao/impl/UserDaoImpl.java]: можно сразу return query.getResultList(); без создания листа

**Review by KostiantynVorobiov — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -8,5 +8,99 @@
...
+                    <configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
+                    <encoding>UTF-8</encoding>
+                    <consoleOutput>true</consoleOutput>
+                    <failsOnError>true</failsOnError>
+                    <linkXRef>false</linkXRef>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
```
Comment (Sofasmile) [inline pom.xml]: do we need this plugin?

File: src/main/java/config/AppConfig.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        properties.put("hibernate.show_sql", "true");
+        properties.put("hibernate.format_sql", "true");
+        properties.put("hibernate.hbm2ddl.auto", "create-drop");
+        factoryBean.setHibernateProperties(properties);
+        factoryBean.setAnnotatedClasses(User.class);
+        return factoryBean;
+    }
+
+    @Bean
+    public Logger getLogger() {
```
Comment (Sofasmile) [inline src/main/java/config/AppConfig.java]: do we need this bean?

Comment (margarita-ignatieva) [reply src/main/java/config/AppConfig.java]: nope, we don't need to inject it anyhow

File: pom.xml (hunk 2 above)
Comment (margarita-ignatieva) [reply pom.xml]: no, not sure, but probably no

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**